TM1py/Services/CellService.py

```diff
@@ -3909,6 +4053,84 @@ def extract_cellset(
             skip_cell_properties=skip_cell_properties,
             skip_sandbox_dimension=skip_sandbox_dimension)
 
+    def extract_cellset_async(
+            self,
+            cellset_id: str,
+            cell_properties: Iterable[str] = None,
+            top: int = None,
+            skip: int = None,
+            delete_cellset: bool = True,
+            skip_contexts: bool = False,
+            skip_zeros: bool = False,
+            skip_consolidated_cells: bool = False,
+            skip_rule_derived_cells: bool = False,
+            sandbox_name: str = None,
+            element_unique_names: bool = True,
+            skip_cell_properties: bool = False,
+            use_compact_json: bool = False,
```

Comment:
looks like "use_compact_json" is not being used

Reply:
resolved with 7992e2a5a2d468a668492155941cd46d8447cbe6